Makefile.dev

```diff
@@ -72,7 +72,7 @@ pluginsopt: $(PLUGINSOPT)
 pluginsbyte: $(PLUGINS)
 
 # This should build all the ocaml code but not (most of) the .v files
-coqocaml: tools coqbinaries $(PLUGINSCMO:.cmo=$(DYNOBJ)) coqide printers bin/votour
+coqocaml: tools coqbinaries $(PLUGINSCMO:.cmo=$(DYNOBJ)) coqide printers bin/votour doc/tools/docgram/fullGrammar
```

Comment:
Can I get an explicit approval from @SkySkimmer that this is fine (given that all this does is to build OCaml files and run an OCaml tool) or, on the contrary, an explicit change request?

Reply:
I meant to remove this.

Reply:
The dependency should be for `world`.